Update profile sync docs to include 3 steps (#4318)
segmentio/segment-docs: src/profiles/profiles-sync/index.md

By default, Segment includes identity graph updates, external ID mapping tables, and two months of the events table in the initial warehouse sync made during setup. Reach out to Segment support if your use case exceeds the scope of the initial setup backfill.

### Step 3: Materialize key views using a SQL automation tool

> To start seeing unified profiles in your warehouse and build attribution models, you'll need to materialize the tables landed by Profile Sync into three key views:

Reviewer: ```suggestion
To start seeing unified profiles in your warehouse and build attribution models, you'll need to materialize the tables that Profiles Sync lands into three key views:
```

>   * `id_graph`:  shows the current state of relationships between segment ids

Reviewer: ```suggestion
  * `id_graph`: the current state of relationships between segment ids
```

>   * `external_id_mapping`: current-state mapping between each external identifier you’ve observed and its corresponding, fully-merged `canonical_segment_id`

Reviewer: ```suggestion
  * `external_id_mapping`: the current-state mapping between each external identifier you’ve observed and its corresponding, fully-merged `canonical_segment_id`
```

  * `profile_traits`: the last seen value for all custom traits, computed traits, SQL traits, audiences, and journeys associated with a profile in a single row

> Please visit the [Tables you materialize](https://segment.com/docs/profiles/profiles-sync/tables/#tables-you-materialize) section of our Profile Sync docs for more inforamation on how to materialize these views either on your own, or with [Segment's open source dbt models](https://github.com/segmentio/profiles-sync-dbt)

Reviewer: ```suggestion
Please visit [Tables you materialize](/docs/profiles/profiles-sync/tables/#tables-you-materialize) for more on how to materialize these views either on your own, or with [Segment's open source dbt models](https://github.com/segmentio/profiles-sync-dbt){:target="blank"}
```
